### PR TITLE
Update Datascience Readme with CSV methodology

### PR DESCRIPTION
**Pull request by brunolajoie**
Updated Datascience Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +88,7 @@ Each country has a GHG mass flow that depends on neighboring countries. In order
   - Gas: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
   - Hydro: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
   - Wind: [ieawind.org](http://www.ieawind.org/countries/norway.html)  
+- Northern Irland: [EIR Grid](http://www.eirgridgroup.com/site-files/library/EirGrid/Generation_Capacity_Statement_20162025_FINAL.pdf)
```
Comment (corradio) [inline README.md]: Spelled Ireland

File: datascience/README.md [hunk 2]
```diff
@@ -112,3 +112,12 @@ Returns :
...
 ```
+
+#### Export as CSV to use in Excel
+Use .to_csv on the dataframe to export it to csv format on your computeur. For instance:
+```python
+from utils import get_exchange
+df_exchange = get_exchange(['DE','FR'], '2016-12-23', '2016-12-24', 1440)
+df_exchange.to_csv('myfilename.csv')
+```
+This should create a new file "myfilename.csv" in the local directory of your computeur from where you launched the jupyter nobook
```
Comment (corradio) [inline datascience/README.md]: Spelled `computer`, but I think you can omit "of your computer"

File: datascience/README.md [hunk 3]
```diff
@@ -112,3 +112,12 @@ Returns :
 3           DE         DK  2016-12-23T00:00:00+00:00    -79.0
 4           DE         CZ  2016-12-23T00:00:00+00:00   -476.0
 ```
+
+#### Export as CSV to use in Excel
```
Comment (corradio) [inline datascience/README.md]: Another way to do it, from the commandline, in one line

```bash
python -c "from utils import get_exchange; get_exchange(['DE','FR'], '2016-12-23', '2016-12-24', 1440).to_csv('myfilename.csv')"
```

File: datascience/README.md [hunk 4]
```diff
@@ -112,3 +112,12 @@ Returns :
...
+Use .to_csv on the dataframe to export it to csv format on your computeur. For instance:
+```python
+from utils import get_exchange
+df_exchange = get_exchange(['DE','FR'], '2016-12-23', '2016-12-24', 1440)
```
Comment (corradio) [inline datascience/README.md]: both lines can be put together, and then you don't need a return object
```python
get_exchange(['DE','FR'], '2016-12-23', '2016-12-24', 1440).to_csv('myfilename.csv')
```

**Comment by corradio** [comment]
Ah sorry. Ignore the comments about the capacities. The changes are there because you did a branch from your capacity branch, and not from master.

**Comment by brunolajoie** [comment]
Damn...i mixed up everything with my branching. sorry

**Comment by corradio** [comment]
It will fix itself once I merge the other PR :)

**Comment by brunolajoie** [comment]
Ok ready to merge on my side

**Pull request merged**